src/web/aws-cloudwatch/CloudWatch.jsx

```diff
@@ -16,6 +18,24 @@ export default class AWSCloudWatch extends Component {
     }).isRequired,
   }
 
+  logOutput = { // Demo Data until API is wired
```

Comment:
The change to include these fields is the Health Check response (#114) has now been merged into the `aws` branch.